.github/workflows/test.yml

```diff
@@ -112,10 +112,3 @@ jobs:
 
       - name: 👀 Run Integration Tests
         run: "yarn test:integration"
-
-      - name: 📤 Upload test results
-        if: always()
-        uses: actions/upload-artifact@v3
-        with:
-          name: playwright-results
-          path: test-results
```

Comment:
What would this do? Would we be able to see the test results somehow? Might be pretty cool to keep if so.

Reply:
on failures, you can have it upload a trace (haven't really looked into that though) and any snapshots that were generated during the test run https://github.com/remix-run/remix/actions/runs/2111669650 look at the bottom for a download link

Reply:
Nice! I think we'll keep off it for now, but maybe we can bring it in later. Thanks!